File: Fascinating Number/Fascinating Number.py
n = 192
l=[]
digits=(str(n)+str(n*2)+str(n*3))
for j in digits:
        l.append(int(j))
        l.sort()
arr=[1,2,3,4,5,6,7,8,9]

if arr==l:
    print ("yes")
else : 
    print ("no")    

File: Fascinating Number/Fascinating Number.cpp
#include <iostream>
#include <string>
#include <algorithm> 
using namespace std;

int main() {
    int n = 42; 
    string s = to_string(n * 2);
    string e = to_string(n * 3);
    string q = to_string(n);
    string v = s + e + q;
    string h = "123456789"; 

    sort(v.begin(), v.end());

    if (h == v) {
        cout << "yes";
    } else {
        cout << "no";
    }
}